plugins/janus_sip.c

```diff
@@ -2023,7 +2100,12 @@ void janus_sip_create_session(janus_plugin_session *handle, int *error) {
 	session->account.outbound_proxy = NULL;
 	session->account.registration_status = janus_sip_registration_status_unregistered;
 	session->status = janus_sip_call_status_idle;
-	session->stack = NULL;
+
+	session->stack = g_malloc0(sizeof(ssip_t));
+	su_home_init(session->stack->s_home);
```

Comment:
I don't think it's a good idea to do this here. IIRC we did it from the loop thread for a reason.

Reply:
It works fine and also fixes a race.